xml/yast2_sw.xml

```diff
@@ -336,21 +336,20 @@
      </mediaobject>
     </informalfigure>
     <para>
-     To perform a search, just write a term in the <guimenu>Package
-     Name</guimenu> input field and press the <guimenu>Search</guimenu> button.
+     To perform a search, just enter a term in the <guimenu>Package
+      Name</guimenu> input field and press <keycap function="enter"/> or the
+     <guimenu>Search</guimenu> button.
      &yast; then contacts the &scc; and, after a few seconds, shows the results
```

Comment:
suggestion to remove "after a few seconds"

Reply:
"A few seconds" may not be correct, but there is indeed a notable delay because the search is performed online by the SUSE Customer Center. Better ideas? "a few moments"?" "after a short moment"?